### DISCUSSION TITLE
Remix deployment issues and reverts

### DISCUSSION THREAD

**Discussion by Shane256eth** [Cyfrin/foundry-full-course-cu#1645] [General]
Hey guys, I'm working in remix on chapter 4.. fundMe contract and for some reason it won't let me use any chain but vm cancun.. it even has a little orange and black button right above the deploy button saying i have to use that chain or it will fail. even when i use that chain almost every function (every function altering state) reverts. cant even fund it. any ideas? Thank you!

**Comment by EngrPips** [comment]
Hello @Shane256eth, I would say maybe close your Remix and re-open it. I have never encounter such issue before.

**Comment by AbigirlMuchineripi** [comment]
hello @Shane256eth, can you kindly share the screenshot and your code maybe I could help.

**Comment by EngrPips** [comment]
Hello @Shane256eth, have You resolved your remix problem already?
